docs/dev/build-instructions/build_linux.md

```diff
@@ -88,6 +88,10 @@ following commands:
 2. `cd build` (Switches into the build directory)
 3. `cmake ..` (Actually runs cmake)
 
+Optionally you can use: `cmake -DRELEASE_ID=$(python "../scripts/mumble-version.py") ..`
+
+This will include the latest commit hash and date in the version info. This should be run in bash, exit fish if you're using it.
```

Comment:
The statement that it should be run in Bash should be enough. I think explicitly mentioning fish here only raises questions as to whether shells other than fish would work.